gobblin-metastore/src/main/java/org/apache/gobblin/metastore/JobHistoryDataSourceProvider.java

```diff
@@ -33,6 +33,7 @@ public class JobHistoryDataSourceProvider extends org.apache.gobblin.util.jdbc.D
 
   @Inject
   public JobHistoryDataSourceProvider(@Named("dataSourceProperties") Properties properties) {
+    super(properties);
```

Comment:
should we worry that perhaps some settings within the base class ctor (like `.setUsername` or `.setPassword`) wouldn't be overridden here, if the config keys used by the base class are present, but the ones used by this derived class are not?

Reply:
line 40 and 41 look like they guard the call

Reply:
missed that, it makes more sense to add the check to this class then